processing/src/test/java/org/apache/druid/java/util/common/guava/ParallelMergeCombiningSequenceTest.java

```diff
@@ -588,7 +588,7 @@ public void testTimeoutExceptionDueToStoppedReader() throws InterruptedException
           INT_PAIR_ORDERING,
           INT_PAIR_MERGE_FN,
           true,
-          1000,
+          5000,
```

Comment:
hmm, this test is supposed to trigger a timeout exception, the sleep doesn't seem long enough to trigger the timeout if we increase this here